.github/workflows/cherry-pick.yml

```diff
@@ -0,0 +1,120 @@
+name: Cherry pick
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - 7.x.x
```

Comment:
Also, out of curiosity, what would this workflow look like once it's merged onto the `main` branch?

Reply:
On the main branch, this workflow would not be triggered. It's intended for the LTS branch to cherry-pick to `main`, so part of setting up an LTS branch would be to update this trigger branch. I'm inclined not to try to set up the workflow in the opposite direction because it would try to cherry-pick changes that might not be intended to go backwards.

Reply:
It can be merged to `main`, it would not be triggered, and it would be updated for a new LTS branch.